backend/src/agents/crew.py

```
@@ -0,0 +1,398 @@
+from crewai import Crew, Agent, Process, Task, LLM
+from pydantic import BaseModel , Field, RootModel
+import os
+import json
+import yaml
+from crewai.project import CrewBase, agent, crew, task
+import logging
+from src.agents.tools.browser_tools import GoToPageTool,FetchAndCleanHTMLTool, GoBackTool, ReloadPageTool, GetCurrentURL, HoverElementTool, SelectDropdownTool, ScrollPageTool, DoubleClickTool, TextDeleteTool, TextInputTool, TakeScreenshotTool, ClickElementTool, FillInputTool
```

Ruff (0.12.2)

`src.agents.tools.browser_tools.TextInputTool` imported but unused

Remove unused import: `src.agents.tools.browser_tools.TextInputTool`

(F401)